packages/core/accounts/kernel/utils/plugins/ep0_7/getEncodedPluginsDataWithoutValidator.ts

```diff
@@ -0,0 +1,48 @@
+import {
+    type Hex,
+    concat,
+    encodeAbiParameters,
+    parseAbiParameters,
+    zeroAddress
+} from "viem"
+import { CALL_TYPE } from "../../../../../constants.js"
+import type { Action } from "../../../../../types/kernel.js"
+
+export const getEncodedPluginsDataWithoutValidator = async ({
```

Comment:
Can just modify `getEncodedPluginsData` to take `enableData` instead of whole validator. Then this extra util isn't required